INSTALL.md

```diff
@@ -128,74 +117,3 @@ EOF
 
 systemctl enable update-binfmt.service
 ```
-
-# Improving Build Speed
-
-## Image Caching and Pruning
-
-Doing a complete build of Rook and the dependent packages can take a long time (more than an hour on a standard laptop). To speed things up we rely heavily on image caching in docker. Docker support content-addressable images by default and we use that effectively when building images. Images are factored to increase reusability across builds. We also tag and timestamp images that should remain in the cache to help future builds.
-
-### Pruning the cache
-
-To prune the number of cached images run `make prune`. There are two options that control the level of pruning performed:
-
-* `PRUNE_HOURS` - the number of hours from when an image was last used (a cache hit) for it to be considered for pruning. The default is 48 hrs.
-* `PRUNE_KEEP` - the minimum number of cached images to keep in the cache. Default is 24 images.
-
-## CI workflow and options
-
-Every PR and every merge to master triggers the CI process in [Jenkins](http://jenkins.rook.io).
-On every commit to PR and master the Jenkins CI will build, run unit tests, and run integration tests.
-If the build is for master or a release, the build will also be published.
-If any of the CI stages fail, then the process is aborted and no artifacts are published.
-On every successful build Artifacts are pushed to an [s3 bucket](https://release.rook.io/). On every successful master build,
-images are pushed to [dockerhub.com](https://cloud.docker.com/u/rook/repository/list).
-
-During the integration tests phase, all tests under [/tests/integration](/tests/integration) are run.
-It may take a while to run all Integration tests. Based on the changes in the PR, it may not be required to run the full CI.
-For faster CI options, add the following tags somewhere in the body of the main PR message.
-
-### [skip ci]
```

Comment:
maybe we can have section on how to `skip ci` for GitHub actions

Reply:
I'm thinking the [skip ci] shouldn't be recommended. All the actions run so fast anyway, it doesn't hurt to just let them all run. When [skip ci] is used, it also disables some of the required actions like codespell and lint that we should run on all commits anyway. Even in draft PRs it can be helpful to still run the actions so we can see what is remaining to be fixed, even if they are expected to fail while in draft.

Reply:
Agreed, we don't have to be resources-cautious like we used to do, too spoiled by those GitHub actions 😅

Reply:
make sense.